taxcalc/utils.py

```diff
@@ -19,6 +21,7 @@
 except ImportError:
     BOKEH_AVAILABLE = False
 
+TAXCALC_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Just wondering about this variable here. What is the intended use case? This variable would not hold a useful value in the case of an installed `taxcalc` package.